drivers/display/ili9340/Kconfig

```diff
@@ -0,0 +1,88 @@
+# Kconfig - ILI9340 display driver configuration options
```

Comment:
don't create dedicated directory unless your really have a lot of dedicated files.

Here have your Kconfig as: 
drivers/display/Kconfig.ili9340